src/environments/environment.stage.ts

```diff
@@ -0,0 +1,4 @@
+export const environment = {
```

Comment:
Where / what is stage env?

Reply:
Stage env is here: https://wiget.gitlab.io/wroclawjug-random-attendee/
This env is used for "pre-prod" tests.